docs/pages/machine-workload-identity/machine-id/deployment/bound-keypair.mdx

```diff
@@ -0,0 +1,174 @@
+---
+
+In this guide, you will install Machine ID's agent, `tbot`, on an arbitrary
+node. This node could be a bare-metal machine, a VM, a container, or any other
+host - the only requirement is that the node has persistent storage.
+
+Note that this is a generic join method. A platform-specific join method may be
+available that is better suited to your environment; refer to the
+[deployment guides](./deployment.mdx) for a full list of options.
+
+## How it works
+
+With Bound Keypair joining, instead of using a static token or relying solely on
+renewable certificates, Machine ID bots generate a unique keypair. Teleport is
+then configured to trust this public key for future joining attempts.
+
+Later, when the bot attempts to join the cluster, Teleport issues it a challenge
+that can only be completed using its private key. The bot returns the solved
+challenge, attesting to its own identity, and is conditionally allowed to join
+the cluster.
+
+As self attestation is inherently less secure than the external verification
+that would be provided by a cloud provider like AWS or a dedicated TPM, bound
+keypair joining enforces a number of additional checks to prevent abuse,
+including:
+- Join state verification to ensure the keypair cannot be usefully shared or
+  duplicated
+- Certificate generation counter checks to ensure regular bot certificates
+  cannot be usefully shared or duplicated
+- Configurable limits on how often - if at all - bots may be allowed to recover
+  using this keypair.
+
+An important benefit to Bound Keypair joining is that all joining restrictions
+can be reconfigured at any time, and bots that expire or go offline can be
+recovered by making a server-side exemption without any client-side
+intervention.
+
+Refer to the [admin guide][guide] for further details on how this join method
+works.
+
+## Prerequisites
+
+{/* note: consider edition-prereqs-tabs.mdx include for v19; it is misleading due to the minor launch release */}
+
+- A running Teleport cluster version 18.1.0 or above.
+- The `tsh` and `tctl` clients
+- (!docs/pages/includes/tctl.mdx!)
+- This guide assumes the bot node has mutable persistent storage for internal
+  bot data. While it is possible to use Bound Keypair joining can on immutable
+  hosts (like CI runs), doing so will reduce security guarantees; see the
+  [admin guide][guide] for further information.
+
+## Step 1/5. Install `tbot`
+
+**This step is completed on the bot node.**
+
+First, `tbot` needs to be installed on the node that you wish to use Machine ID
+on.
+
+Download and install the appropriate Teleport package for your platform:
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 2/5. Create a Bot
+
+**This step is completed on your local machine.**
+
+(!docs/pages/includes/machine-id/create-a-bot.mdx!)
+
+## Step 3/5. Create a join token
+
+**This step is completed on your local machine.**
+
+In this guide, we'll demonstrate joining a bot using a registration secret: this
```

Comment:
I would mention both approaches before we narrow down on one, just so the reader knows there's another option.

Maybe:

> The initial joining operation can operate in two modes. The first is by using
> a registration secret, which is similar to the first join process with the `token`
> join method. Alternatively, initial joining can be performed with a public key.
>
> In this guide, we will demonstrate joining using a registration secret.

Reply:
I've added an explicit reference (and a link) in the admonition, hopefully it makes the existence of the alternative clear? Trying to balance clarity with the guide only actually showing how to accomplish a single mode.

(Maybe showing both paths in the guide will be more reasonable with a `tctl` helper - may update this further if so)